deepchem/models/torch_models/hf_models.py

```diff
@@ -0,0 +1,407 @@
+import logging
+import os
+import time
+from collections.abc import Sequence as SequenceCollection
+from typing import (TYPE_CHECKING, Any, Callable, Iterable, List, Optional,
+                    Tuple, Union)
+
+import numpy as np
+import torch
+from deepchem.models.optimizers import LearningRateSchedule
+from deepchem.models.torch_models import TorchModel
+from deepchem.trans import Transformer, undo_transforms
+from deepchem.utils.typing import LossFn, OneOrMany
+from transformers.data.data_collator import DataCollatorForLanguageModeling
+
+logger = logging.getLogger(__name__)
+
+if TYPE_CHECKING:
+    import transformers
+    from transformers.modeling_utils import PreTrainedModel
+
+
+class HuggingFaceModel(TorchModel):
+    r"""HuggingFace model wrapper
+
+    The class provides a wrapper for wrapping models from the `HuggingFace
+    ecosystem in DeepChem and training it via DeepChem's api.
+
+    Parameters
+    ----------
+    models: transformers.modeling_utils.PreTrainedModel
+        The HuggingFace model to wrap.
+    task: str
+        Pretraining or finetuning task
+    tokenizer: transformers.tokenization_utils.PreTrainedTokenizer
+        Tokenizer
+
+   .. code-block:: python
```

Comment:
This should be a doctest instead of a code block

Reply:
made it as a doctest